com.unity.render-pipelines.core/Runtime/Lighting/ProbeVolume/ProbeReferenceVolume.Debug.cs

```diff
@@ -346,7 +346,7 @@ CellInstancedDebugProbes CreateInstancedProbes(CellInfo cellInfo)
 
             var cell = cellInfo.cell;
 
-            if (!cell.shL0L1Data.IsCreated || cell.shL0L1Data.Length == 0 || !cellInfo.loaded)
+            if (!cell.bricks.IsCreated || cell.bricks.Length == 0 || !cellInfo.loaded)
```

Comment:
because it should be the same but now shL0L1Data can be inside state0 or state1.
Also bricks is accessed from c# here whereas shL0L1Data is accessed in shader but should not changed much